app.js

```diff
@@ -34,13 +34,15 @@ function initApp(config, callback) {
 	};
 
 	async.series([
-
 		function(next) {
 			/* eslint camelcase: 'off' */
-			MongoClient.connect(config.database, {server: {auto_reconnect: false}}, function(error, db) {
+			MongoClient.connect(config.database, {
+				autoReconnect: true
+			}, function(error, db) {
 				if (error) {
 					console.log('Error connecting to MongoDB:');
 					console.log(JSON.stringify(error));
+					return next(error);
```

Comment:
will this just keep trying to reconnect forever?

Reply:
Nope. The `next` callback has signature `next(error, value)`, so passing in just an error stops the whole thing.

And the `return` is just a pattern to exit there. It's synonmous to:
```
next(error);
return;
```